x-pack/legacy/plugins/siem/public/components/timeline/timeline.tsx

```diff
@@ -170,7 +170,14 @@ export const Timeline = React.memo<Props>(
                   getUpdatedAt,
                   refetch,
                 }) => (
-                  <TimelineRefetch loading={loading} id={id} inspect={inspect} refetch={refetch}>
+                  <TimelineRefetch
```

Comment:
While desk-testing, I'm seeing a timeline-specific (the `Hosts` and `Network` tabs behave as expected) issue where the timeline only updates when you click `Refresh` if there is already data. To reproduce:

1.  Click the `SIEM` lock icon to force a full page refresh. The `Overview` page will become the active tab.
2. Click the timeline flyout button to open the timeline.
3. In the timeline KQL bar, type `host.name : *`, but do NOT press enter.
4. Click the `Refresh` button in the timeline.

Expected result:
- The timeline updates

Actual Result
- The timeline does not update, no matter how many times `Refresh` is clicked

5. Re-focus input to the KQL bar, then press enter. The timeline now shows data as expected.
6. Note the hostnames shown in the `host.name` column.
7. Change the KQL query from `host.name : *` to something that mentions a specific hostname, e.g. `host.name : siem-kibana`, but do NOT press enter.
8. Click the `Refresh` button in the timeline. The timeline now updates as expected, showing only the hostnames that match the KQL query.